src/Listener/Symfony/MessengerListener.php

```diff
@@ -28,9 +28,9 @@ public function onMessageReceived(WorkerMessageReceivedEvent $event): void
             if (is_object($message) && !in_array($class = get_class($message), $this->config->getMessageToAssert(), true)) {
                 $this->logger->log(
                     $this->logLevel,
-                    'Untracked {message} has been detected, add it in your configuration to ensure ZDD compliance.',
+                    sprintf('Untracked message "%s" has been detected, add it in your configuration to ensure ZDD compliance.', $class),
```

Comment:
Keep using PSR-3 placeholders please. That's more flexible :pray:

Reply:
See https://github.com/Yousign/zdd-message-bundle/pull/24#discussion_r1350189598

Reply:
If we do this the key in context will be named `message` and it will be used by our log tool as the error message isn't it?

Reply:
Nop we can use any variable we want with PSR-3 placeholders. Here is what you can test:

```suggestion
                    'Untracked message "{class}" has been detected, add it in your configuration to ensure ZDD compliance.',
```

Reply:
Yes, see the last version and THANKS!